### DISCUSSION TITLE
what is difference between data.ms , meili_data

### DISCUSSION THREAD

**Discussion by vahidalvandi** [meilisearch/product#724] [Feedback & Feature Proposal]
i have issue in import dump in docker .
after import dump directory  **data.ms** contain index imported but in meili_data index directory is empty!! and in  Mini Dashboard dont show indexes

**Comment by irevoire** [comment]
Hello, can you show the commands you ran?

> what is difference between data.ms , meili_data

The `data.ms` contains the database.
`meili_data` contains three things:
- The `data.ms` directory -> with the database
- The `dumps` directory, which contains all the dumps you create
- The `snapshots` directory which contains the snapshots you create

**Comment by vahidalvandi** [comment]
## Export from Docker

`docker exec -it container-meilisearch-1 sh
`
```
curl \
  -X POST 'http://localhost:7700/dumps' \
  -H 'Authorization: Bearer zsdaASDAScvbhdasdsfljksmcask1dsza'
```

move dump file 20240221-055247195.dump in folder

## Import from Docker


`docker exec -it docker-containers-meilisearch-1 sh
`cd /

#remove old data.ms directory
`rm -rf data.ms/`

`./meilisearch --import-dump /meili_data/dumps/20240221-055247195.dump --master-key="ascasc12sdSADf"
`

**Reply by irevoire** [reply]
Hey, your issue is that you didn’t use a volume with docker.
This means that your disk got reset between your first and second commands.

As you can see in our documentation, we recommend you to use a volume to avoid this kind of issue: https://www.meilisearch.com/docs/learn/cookbooks/docker#run-meilisearch-with-docker
